packages/algorand-js/src/lib/mocks.ts

```diff
@@ -0,0 +1,25 @@
+import { TxParams } from "@algorand-builder/algob/src/types";
+import { SuggestedParams } from "algosdk";
+
```

Comment:
Please move this file to `/src/mock/tx.ts`

Reply:
ok